Review user guide (#979)
ansible/receptor: docs/source/user_guide/k8s.rst

kubeitpod.yml

> N.B. it may necessary to have one of tcp-listener, tcp-peer, or local-only to be able to start the control service. See https://github.com/ansible/receptor/issues/518

Reviewer: We shouldn't use N.B. here.  You can turn this into an RST note directive to get better highlighting. See https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html

Author: Agreed, changed to note

kubeitpod.yml

.. note::

>    It may be necessary to have one of tcp-listener, tcp-peer, or local-only to be able to start the control service. See https://github.com/ansible/receptor/issues/518

Reviewer 2: ```suggestion
   It may be necessary to have ``tcp-listener``, ``tcp-peer``, or ``local-only`` to start the control service. See https://github.com/ansible/receptor/issues/518
```

Author: combined with a prior suggestion

>    It may be necessary to have one of tcp-listener, tcp-peer, or local-only to be able to start the control service. See https://github.com/ansible/receptor/issues/518

Reviewer 3: Active voice:
```suggestion
   You may need to run one of tcp-listener, tcp-peer, or local-only before you can start the control service. See https://github.com/ansible/receptor/issues/518.
```
